frame/staking/src/pallet/impls.rs

```diff
@@ -783,8 +783,14 @@ impl<T: Config> Pallet<T> {
 				None => break,
 			};
 
+			let voter_weight = weight_of(&voter);
+			// if voter weight is zero, do not consider this voter for the snapshot.
+			if voter_weight.is_zero() {
+				log!(warn, "voter's active balance is 0. skip this voter.");
```

Comment:
We will know this will cause warnings on Kusama, which all nodes will see. Should be a debug log or something that is not visible? Otherwise it is a false alarm for eg. validators.

Reply:
Changed the log target to debug.